### PR TITLE
Update Privacy Pro translations

### PR DESCRIPTION
**Pull request by miasma13**
Task/Issue URL: https://app.asana.com/0/0/1208750072186506/f

**Description**:
Update Privacy Pro and VPN related translations.

**Optional E2E tests**:
- [ ] Run PIR E2E tests
	Check this to run the Personal Information Removal end to end tests. If updating CCF, or any PIR related code, tick this.

**Steps to test this PR**:
Change preferred language for the app and verify completeness for:
- Privacy Pro settings when not signed in
- Privacy Pro settings when signed in
- VPN settings and popover views

**Definition of Done**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)

### CODE REVIEW

File: DuckDuckGo/Localizable.xcstrings [hunk 1]
```diff
@@ -8344,44 +8344,44 @@
         },
         "es" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "The following Bitwarden versions are incompatible with DuckDuckGo: v2024.3.0, v2024.3.2, v2024.4.0, v2024.4.1. Please update to a newer version by following these steps:"
+            "state" : "translated",
+            "value" : "The following Bitwarden versions are incompatible with DuckDuckGo: v2024.10.0, v2024.10.1, v2024.10.2. Please downgrade to an older version by following these steps:"
```
Comment (SabrinaTardio) [inline DuckDuckGo/Localizable.xcstrings:8348]: I am not sure about this, can you check with Tom?

**Review by SabrinaTardio — COMMENTED** [review]
For new Submodules, (and potentially targets) add English US to the list of languages and translate one string.
For SyncUI we noticed that english was defaulting to another language and doing this fixed it

Some String from a catalogue and some full catalogues are still missing translation but I assume this are still ongoing

File: DuckDuckGo/Localizable.xcstrings (hunk 1 above)
Comment (miasma13) [reply DuckDuckGo/Localizable.xcstrings:8348]: Looks correct, these were update on Nov 1 in 4de7c6c

**Comment by miasma13** [comment]
PR closed due to old release branch being deleted. Reopened in https://github.com/duckduckgo/macos-browser/pull/3656

**Pull request closed without merging**